core-bundle/src/Resources/config/services.yml

```diff
@@ -493,6 +493,19 @@ services:
             - '%contao.prepend_locale%'
         public: true
 
+    Contao\CoreBundle\Search\Indexer\DefaultIndexer:
+        arguments:
+            - '@contao.framework'
+            - '@database_connection'
+        tags:
+            - { name: 'contao.search_indexer' }
+
+    Contao\CoreBundle\Search\Indexer\DelegatingIndexer: ~
```

Comment:
```suggestion
    contao.search.delegating_indexer: ~
```

Reply:
No we agreed to use class names for new services.

Reply:
Fixed.